neonvm/config/default-vxlan/network/overlay-for-pods.yaml

```diff
@@ -1,17 +1,18 @@
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: overlay-net-ipam
+  name: overlay-for-pods
   namespace: system
 spec:
   config: '{
     "cniVersion": "0.3.1",
-    "name": "neonvm-overlay-net-ipam",
+    "name": "overlay-for-pods",
     "type": "bridge",
     "bridge": "neon-br0",
     "ipam": {
       "type": "whereabouts",
-      "range": "10.100.0.0/20",
-      "range_start": "10.100.0.100"
-      }
-    }'
+      "range": "10.100.0.0/16",
+      "range_start": "10.100.1.0",
+      "range_end": "10.100.127.255"
+    }
+  }'
```

Comment:
Overlay networking still not used in staging and prod, so no problem there

Reply:
I was more concerned that we could have problems because the the configs overlap during upgrade, so maybe we have to delete the old one before creating the new one. But I tried it locally, and can see that the new network-attachment-definition is created ok even with the old one there.

And given we're not actually using it, should be fine, right? (one last double-check that I'm understanding this correctly)

Reply:
> manually remove old definitions after deploy

I've already been doing that with previous updates. Is there anything different required here?